### PR TITLE
{data,geo}[foss/2020b,intel/2020b] Cartopy v0.19.0.post1, Fiona v1.8.20, Shapely v1.8a1 w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by deniskristak**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/c/Cartopy/Cartopy-0.19.0.post1-foss-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    ('Shapely', '1.8a1'),
+    ('lxml', '4.6.2'),
+    ('Pillow', '8.0.1'),
+    ('PROJ', '7.2.1'),
+    ('PyYAML', '5.3.1'),
+]
+
+use_pip = True
+sanity_pip_check = True
+download_dep_fail = True
```
Comment (boegel) [inline easybuild/easyconfigs/c/Cartopy/Cartopy-0.19.0.post1-foss-2020b.eb]: `download_dep_fail` is enabled by default for `PythonBundle`, so please remove (required to make easyconfigs test suite pass)

File: easybuild/easyconfigs/c/Cartopy/Cartopy-0.19.0.post1-intel-2020b.eb [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    ('Shapely', '1.8a1'),
+    ('lxml', '4.6.2'),
+    ('Pillow', '8.0.1'),
+    ('PROJ', '7.2.1'),
+    ('PyYAML', '5.3.1'),
+]
+
+use_pip = True
+sanity_pip_check = True
+download_dep_fail = True
```
Comment (boegel) [inline easybuild/easyconfigs/c/Cartopy/Cartopy-0.19.0.post1-intel-2020b.eb]: drop, enabled by default

File: easybuild/easyconfigs/f/Fiona/Fiona-1.8.20-foss-2020b.eb [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('GDAL', '3.2.1'),
+    ('Shapely', '1.8a1'),
+]
+
+use_pip = True
+download_dep_fail = True
+sanity_pip_check = True
```
Comment (boegel) [inline easybuild/easyconfigs/f/Fiona/Fiona-1.8.20-foss-2020b.eb:24]: already enabled below, remove duplicate

File: easybuild/easyconfigs/f/Fiona/Fiona-1.8.20-foss-2020b.eb [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('GDAL', '3.2.1'),
+    ('Shapely', '1.8a1'),
+]
+
+use_pip = True
+download_dep_fail = True
```
Comment (boegel) [inline easybuild/easyconfigs/f/Fiona/Fiona-1.8.20-foss-2020b.eb]: drop, enabled by default

File: easybuild/easyconfigs/f/Fiona/Fiona-1.8.20-intel-2020b.eb [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+toolchain = {'name': 'intel', 'version': '2020b'}
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('GDAL', '3.2.1'),
+    ('Shapely', '1.8a1'),
+]
+
+use_pip = True
+download_dep_fail = True
```
Comment (boegel) [inline easybuild/easyconfigs/f/Fiona/Fiona-1.8.20-intel-2020b.eb]: drop, enabled by default

File: easybuild/easyconfigs/f/Fiona/Fiona-1.8.20-intel-2020b.eb [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+sanity_pip_check = True
```
Comment (boegel) [inline easybuild/easyconfigs/f/Fiona/Fiona-1.8.20-intel-2020b.eb:24]: duplicate, keep the one below

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/c/Cartopy/Cartopy-0.19.0.post1-foss-2020b.eb (hunk 1 above)
Comment (deniskristak) [reply easybuild/easyconfigs/c/Cartopy/Cartopy-0.19.0.post1-foss-2020b.eb]: yep

File: easybuild/easyconfigs/f/Fiona/Fiona-1.8.20-foss-2020b.eb (hunk 3 above)
Comment (deniskristak) [reply easybuild/easyconfigs/f/Fiona/Fiona-1.8.20-foss-2020b.eb:24]: yes

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
node3513.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/855f133267da2ac8762de9f4f25a7e0b for a full test report.

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
node2613.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/d88d9162d954882e27f1df2ec73e2748 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @deniskristak!

**Pull request merged**